static/src/javascripts/projects/common/modules/cmp-ui/index.js

```diff
@@ -31,6 +33,7 @@ export const init = (forceModal: boolean) => {
                 "'Guardian Text Sans Web', Helvetica Neue, Helvetica, Arial, Lucida Grande, sans-serif",
         },
         forceModal,
+        variant: isInVariantSynchronous(commercialCmpCopy, 'variant') ? 'commercialCmpCopy-variant' : 'commercialCmpCopy-control',
```

Comment:
The reader not being in the commercialCmpCopy variant group does not mean they are in the control group. This parameter should still be empty if they are not part of the AB test at all. This is important because it gets reported to the consent logs and that's how D&I will make the distinction between control and variant groups.